models/teams.js

```diff
@@ -0,0 +1,13 @@
+const Sequelize = require('sequelize')
```

Comment:
We're passing in the library when we call this function in `index.js` so no need to import it here. Looks like your parameter casing may have tripped you up. The `sequelize` in `(connection, sequelize) => {` is the library being imported in `index.js`